official-antistasi-community/A3-Antistasi
# Update System Mod detection to A3A Preset / TFAR Beta detection

## What type of PR is this.
1. [x] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added the A3A_ Preset to System Mods in InitVarCommon,
Added TFAR Beta Detection in InitVarCommon,
Added a Check that returns TFAR Stable as False when Beta is detected,
Added A3A_HasTFARBeta to all Cases which would not cause Errors,
Added TFARBeta Radios to Rebell Templates.

### Please specify which Issue this PR Resolves.
closes #1604 

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Quick general check:
A3A_HasTFARBeta should return true when loaded,
A3A_HasTFAR should return false when Beta is loaded.
Try to run the Mission with TFAR Beta,
Beta items are given to Starting Items,
Other BackpacksRadio are apearing in Loot (allBackpacksRadio will return all of them),

********************************************************
Notes:
